app/helpers/repository_helper.rb

```diff
@@ -23,8 +23,7 @@ def month_options
   end
 
   def year_options
-    # FIXME: this will not work some years from now
-    (2013..2020).to_a.map {|year| [year, year]}
+    (DateTime.now.year..DateTime.now.year + 5).to_a.map {|year| [year, year]}
```

Comment:
I'm sorry about this. I just thought the goal was only the years that were to come, haha. Now what you are talking make all sense. The goal is to restrict future years, not otherwise.